pkg/secrets/k8s/k8s_secrets.go

```diff
@@ -1,6 +1,7 @@
 package k8s
 
 import (
+	"bytes"
```

Comment:
please sort this according to this commit (https://github.com/cyberark/conjur-authn-k8s-client/pull/47/commits/ef46d83efe120cf5d4f537c0f185427e5a242d61)